roboteam_ai/src/ApplicationManager.cpp

```diff
@@ -98,11 +98,23 @@ void ApplicationManager::runOneLoopCycle() {
         }
 
 
+        ros::Time begin;
+        ros::Time end;
+        if (ai::interface::Output::showCoachTimeTaken()) {
+            begin = ros::Time::now();
+        }
+
         rtt::ai::coach::getBallCoach->update();
         rtt::ai::coach::g_DefenceDealer.updateDefenderLocations();
         rtt::ai::coach::g_offensiveCoach.updateOffensivePositions();
         rtt::ai::coach::g_pass.updatePassProgression();
 
+        if (ai::interface::Output::showCoachTimeTaken()) {
```

Comment:
I don't like logging under the condition of checkboxes since it clutters the interface

Reply:
it's currently not in interface.